src/weather.c

```diff
@@ -59,6 +60,20 @@ static char* weather_conditions[] = {
     "\U0000F073", // hurricane: 46
 };
 
+const int very_stale_weather_threshold = 121;
+const int stale_weather_threshold = 31;
+
+void update_expired_weather(int current_time) {
+  if(!current_time)
+    current_time = (int)time(NULL);
+  if(current_time - last_successful_update >= very_stale_weather_threshold * 60)
+    set_weather_layer_color(GColorFromHEX(0xFFAAAA));
```

Comment:
Were you able to test how this will look on monochrome pebbles?

Reply:
Nope; I only have a Pebble time, and as implied, this was a quick hack to make it work for me.